<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>CCC</string>
   </property>
   <property name="statusTip">
    <string>wwww</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>we </string>
   </property>
   <property name="placeholderText">
    <string>2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="ResultBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>104</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
